agent/tool-scripts/postprocess/gold/jmap/csv/jmap.csv

```diff
@@ -0,0 +1,4 @@
+timestamp,532,java-1,java-2
+1526474998000,131.001
+1526474998000,,131.696
+1526474998000,,,131.000
```

Comment:
Shall we set the empty values to 0 or is it fine to leave it as is? @portante @ndokos thoughts?

Reply:
as far as I test as empty it works with no issue and graphs are fine.

Reply:
I don't understand the format: why are there two fields in some cases, three or four (including empty fields) in other cases?

Reply:
right, thats upon to the jschart valid formats..
https://github.com/distributed-system-analysis/pbench/blob/master/web-server/v0.3/js/jschart.js#L238

every column represents a process id or name.
you can find similar format on disk tool (there, we have timestamp and each column represents device).

from the tests I've done so far this format is valid and the graphs are reliable and pass all tests.

In this csv I've included process-name and id as example
532 is pid
java1 is process name